jekyll/_docs/build-image-trusty.md

```diff
@@ -3,11 +3,14 @@ layout: classic-docs
 title: Ubuntu 14.04 (Trusty)
 categories: [build-images]
 description: Ubuntu 14.04 (Trusty)
+changefreq: "weekly"
 ---
 
-You can run your Linux builds on Ubuntu 14.04 Trusty (default is Ubuntu 12.04). You can switch to Trusty from "Project Settings" -> "Build Environment" of your project.
+You can run your Linux builds on Ubuntu 14.04 Trusty (the default). You can 
```

Comment:
Change to:

You can run your Linux build on the default, Ubuntu 14.04 Trusty. You can switch to Trusty from "Project Settings" -> "Build Environment" of your project.